platform/keysystem/worldconfig/keyclient.go

```diff
@@ -51,145 +59,188 @@ func GetVariant() (string, error) {
 	return kind, nil
 }
 
-func GetDownloads(variant string) []config.ConfigDownload {
-	downloads := []config.ConfigDownload{
-		{
+func (b *ActionBuilder) DefaultDownloads(variant string) {
+	b.Download(config.ConfigDownload{
+		Type:    "authority",
+		Name:    "kubernetes",
+		Path:    "/etc/homeworld/authorities/kubernetes.pem",
+		Refresh: "24h",
+		Mode:    "644",
+	})
+	b.Download(config.ConfigDownload{
+		Type:    "authority",
+		Name:    "clustertls",
+		Path:    "/usr/local/share/ca-certificates/extra/cluster.tls.crt",
+		Refresh: "24h",
+		Mode:    "644",
+	})
+	b.Download(config.ConfigDownload{
+		Type:    "authority",
+		Name:    "ssh-user",
+		Path:    "/etc/ssh/ssh_user_ca.pub",
+		Refresh: "168h", // allow a week for mistakes to be noticed on this one
```

Comment:
OOC can you explain this comment?

Reply:
I don't remember exactly, but it's something like "ssh is very important and so we should update it less frequently, so that if things break badly, the ssh keys previously generated are more likely to still work when we realize things are broken"

Reply:
I think the idea is that everything else has a shorter delay, so breakages would be likely to occur in non-SSH components first, whether in dev or prod, and we'd notice breakage through that first.
(In any case, this PR just preserves the existing behavior, so we should take this to a new issue if we think we should change it.)